libcontainer/cgroups/fs/blkio_test.go

```diff
@@ -210,23 +211,24 @@ func TestBlkioSetWeightDevice(t *testing.T) {
 	wd := configs.NewWeightDevice(8, 0, 500, 0)
 	weightDeviceAfter := wd.WeightString()
 
+	blkio := &BlkioGroup{}
+	blkio.detectWeightFilenames(helper.CgroupPath)
```

Comment:
Please note we are not using real cgroupfs here, but a mock of it, so this code won't work the same way (instead, if you pre-create `blkio.weight`, it will be used, otherwise BFQ). Maybe it makes sense to test both scenarios (with and without `blkio.weight` pre-created by the test).

Reply:
Fixed.